lib/money-rails/mongoid/money.rb

```diff
@@ -33,6 +29,8 @@ def mongoize(object)
       return nil if object.nil?
       return mongoize_castable(object) if object.respond_to?(:to_money)
 
+      # QUESTION: why whould we even want to store it?
```

Comment:
You might wanna double check this, but I think this is the way Mongoid expects it to behave (I think I saw some other examples of this). I agree that it seems weird that you'd just use whatever has been given to you.

In case that's not true, I'm happy to return `nil` or raise here